app/routes/app._index.tsx

```diff
@@ -29,7 +29,7 @@ export const action = async ({ request }: ActionFunctionArgs) => {
   ];
   const response = await admin.graphql(
     `#graphql
-      mutation populateProduct($input: ProductInput!) {
+      mutation shopifyRemixTemplatePopulateProduct($input: ProductInput!) {
```

Comment:
Is it possible not to change this mutation name? Or it makes more sense to change this? We are currently using any `populateProduct` as automatic calls given that any `populateProduct` can be considered as automatic calls. What do you think?